src/Popup/SystemUseWarningPopup.jsx

```diff
@@ -3,29 +3,24 @@ import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import Popup from '../components/Popup';
 import { components } from '../params';
-import { hostname } from '../localconf';
+import { hostnameWithSubdomain } from '../localconf';
 import { updateSystemUseNotice } from '../actions';
 
 const handleAcceptWarning = () => {
   /**
    * If policy is accepted then set systemUseWarnPopup to false, hiding the
    * warning.
    */
-  // set a new cookie indicating we have accepted this policy
-
-  const hostnameParts = hostname.split('.');
-  const hLen = hostnameParts.length;
-  const hostnameNoSubdomain = (hLen > 2) ? hostnameParts.splice(hLen - 2).join('.') : hostname;
-  const domain = hostnameNoSubdomain.endsWith('/') ? hostnameNoSubdomain.slice(0, -1) : hostnameNoSubdomain;
```

Comment:
for example, with `hostname='https://healdata.org'`, the value of `domain` from above code will be `https://healdata`, which is incorrect